src/routes/user.js

```diff
@@ -260,7 +263,10 @@ router.get('/verification/:token', verifyUser);
 
 router.post('/logout', logout);
 
+router.post('/resetpassword', resetPassword);
 
+router.put('/resetpassword/new', createNewPassword); 
```

Comment:
Plz since you're update you don't need to add this `new` just `resetpassword` is enough. For the best practice consider removing that.